### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by spartucus**
Refactoring code by remove unused variables.

### CODE REVIEW

File: src/ringct/multiexp.cc [hunk 1]
```diff
@@ -447,15 +446,14 @@ rct::key straus(const std::vector<MultiexpData> &data, const std::shared_ptr<str
 {
   CHECK_AND_ASSERT_THROW_MES(cache == NULL || cache->size >= data.size(), "Cache is too small");
   MULTIEXP_PERF(PERF_TIMER_UNIT(straus, 1000000));
-  bool HiGi = cache != NULL;
+  bool cache != NULL;
```
Comment (iamamyth) [inline src/ringct/multiexp.cc]: Looks like this shouldn't even compile.

Comment (vtnerd) [reply src/ringct/multiexp.cc]: This is overriding the existing variable, so a change is desireable either way.

Comment (spartucus) [reply src/ringct/multiexp.cc]: fix it now, remove the whole line.

**Comment by moneromooo-monero** [comment]
This looks like old code, this HiGi line was removed months ago (others might still be valid, I did not check).

**Pull request closed without merging**